docs/guides/preparing/local-server-setup.mdx

```diff
@@ -0,0 +1,302 @@
+```
+
+You also need to mark the script file as executable. Start by typing the following command below into your terminal.
+
+```bash
+chmod +x <path to your start.sh file>
+```
+
+You must always execute from `start.sh` to launch the server, not the .jar file.
+
+### Downloading PGM
+
+You have two options when it comes to downloading PGM. You can download the stable version of PGM from the [Downloads](/downloads) page or you can download the development builds from its [GitHub repository](https://github.com/PGMDev/PGM/actions?query=actor%3AElectroid+branch%3Adev+is%3Asuccess).
+If you are working on a map that will utilize features introduced in pre-release versions of PGM, we recommend downloading the development build.
+
+Once the download is done, you must put `PGM.jar` inside the `plugins` folder (create if it does not exist), which can be found in the folder that contains the `SportPaper.jar` file.
+
+### Starting the Server
+
+Once you are done with the initial setup, you can launch from your terminal (`./start.sh`) or double-click `start.sh` (if using a desktop environment) to let SportPaper create the necessary files for Minecraft and PGM.
+You can now join the server from your Minecraft client at `localhost` or `127.0.0.1:25565`.
+
+Also see [Additional Setup](#additional-setup) if you would like to make the most use out of your newly-created server.
+
+## Additional Setup
+
+### Configure SportPaper
+
+Using your text editor of choice, you may edit `server.properties` and `sportpaper.yml`. We recommend changing the following options below:
+
+```properties title="server.properties"
+allow-nether=false
+announce-player-achievements=false
+generate-structures=false
+```
+
+```yml title="sportpaper.yml"
+bukkit:
+  settings:
+    allow-end: false
+spigot:
+  # Forcibly disable vanilla Minecraft achievements.
+  stats:
+    disable-saving: true
+    forced-stats:
+      achievement:
+        openInventory: 1
+  world-settings:
+    anti-xray:
+      enabled: false # Helps with reducing lag; public servers should utilize an anti-cheat plugin.
+paper:
+  settings:
+    empty-server-suspend: P100D # Prevent the server from immediately suspending itself due to inactivity.
+  world-settings:
+    fix-cannons: true # Recommended when making maps that involves TNT.
+    game-mechanics:
+      disable-chest-cat-detection: true # Helps with reducing lag.
+    optimize-explosions: true # Recommended when making maps that involves TNT.
+```
+
+### Configure PGM
+
+PGM will load with five default maps out of the box. 
+You are expected to configure the plugin before you can begin using additional maps.
+We encourage you to understand and modify PGM's config, similarly to what we've done below:
+
+```yml title="/plugins/PGM/config.yml"
+map:
+  folders:
+    # List of folders to load maps.
+    # You can add multiple folders from different sources.
+    - "sample-github-maps"
```

Comment:
you do not need the same map defined as both a folder, and a git repo, it could lead to confusion if both are in the config